include/wx/unichar.h

```diff
@@ -267,8 +267,8 @@ class WXDLLIMPEXP_BASE wxUniCharRef
     wxUniCharRef& operator=(const wxUniCharRef& c)
         { if (&c != this) *this = c.UniChar(); return *this; }
 
-#if __cplusplus >= 201103
-    wxUniCharRef(const wxUniCharRef&) = default;
+#ifdef wx_MEMBER_DEFAULT
```

Comment:
This looks like a typo?
```suggestion
#ifdef wxMEMBER_DEFAULT
```